interface/forms/ros/C_FormROS.class.php

```diff
@@ -1,30 +1,47 @@
 <?php
-
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * Sherwin Gaddis <sherwingaddis@gmail.com>
```

Comment:
I finally got a chance to get back to this form and I have looked at some of the new code. Here is my two cents. 

In the smarty template there is a line that is 
 {php}html_header_show();{/php}
which as you know sets the theme that is going to be used. My question is there a way to deliver the bootstrap location the same way? That way it is universally distributed like the theme. Then we all should be able to build or rebuild on the same framework. 
Am I talking out of my head or saying what is already in the works?